Code/Framework/AzFramework/Platform/Linux/platform_linux.cmake

```diff
@@ -5,3 +5,30 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 #
 #
+
+# Based on the linux window manager trait, perform the appropriate additional build configurations
+# Only 'xcb', 'wayland', and 'xlib' are recognized
+if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
+
+    find_library(XCB_LIBRARY xcb)
+
+    set(LY_BUILD_DEPENDENCIES
+        PRIVATE
+            ${XCB_LIBRARY}
+    )
+
+    set(LY_COMPILE_DEFINITIONS PUBLIC AZ_LINUX_WINDOW_MANAGER_XCB)
+
+elseif(PAL_TRAIT_LINUX_WINDOW_MANAGER STREQUAL "wayland")
+
+    set(LY_COMPILE_DEFINITIONS PUBLIC AZ_LINUX_WINDOW_MANAGER_WAYLAND)
+
+elseif(PAL_TRAIT_LINUX_WINDOW_MANAGER STREQUAL "xlib")
+
+    set(LY_COMPILE_DEFINITIONS PUBLIC AZ_LINUX_WINDOW_MANAGER_XLIB)
```

Comment:
These compile definitions should use the trait naming scheme for consistency with other AzFramework traits.
i.e `AZ_TRAIT_AZFRAMEWORK_LINUX_WINDOW_MANAGER_XCB`

Reply:
Done